packages/x-components/src/x-modules/queries-preview/components/query-preview.vue

```diff
@@ -151,6 +152,13 @@
     @Inject({ default: undefined })
     protected location?: FeatureLocation;
 
+    /**
+     * Query Preview key converted into a unique id.
+     *
+     * @internal
+     */
+    public queryOfQueryPreview = getHashFromQueryPreviewInfo(this.queryPreviewInfo);
```

Comment:
I would rename it from `queryOfQueryPreview` to something like `queryOfQueryPreviewHash` or similar so it is clear that we are dealing with a hash.